ext/simplexml/simplexml.stub.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/** @return SimpleXMLElement */
+function simplexml_load_file(string $filename, string $class_name = 'SimpleXMLElement', int $options = 0, string $ns = null, bool $is_prefix = false) {}
+
+/** @return SimpleXMLElement */
+function simplexml_load_string(string $data, string $class_name = 'SimpleXMLElement', int $options = 0, string $ns = null, bool $is_prefix = false) {}
+
+/** @return SimpleXMLElement */
+function simplexml_import_dom(DOMNode $node, string $class_name = 'SimpleXMLElement') {}
+
+class SimpleXMLElement {
+    /** @return array|false */
+    public function xpath(string $path) {}
+
+    /** @return bool */
+    public function registerXPathNamespace(string $prefix, string $ns) {}
+
+    /** @return string|bool */
+    public function asXML(string $filename = null) {}
+
+    /** @return string|bool */
+    public function saveXML(string $filename = null) {}
+
+    /** @return array */
+    public function getNamespaces(bool $recursve = false) {}
+
+    /** @return array|false */
+    public function getDocNamespaces(bool $recursve = false, bool $from_root = true) {}
+
+    /** @return SimpleXMLIterator */
+    public function children(string $ns = null, bool $is_prefix = false) {}
+
+    /** @return SimpleXMLIterator */
+    public function attributes(string $ns = null, bool $is_prefix = false) {}
+
+    public function __construct(string $data, int $options = 0, bool $data_is_url = false, string $ns = null, bool $is_prefix = false) {}
+
+    /** @return SimpleXMLElement */
+    public function addChild(string $name, string $value = null, string $ns = null) {}
+
+    /** @return SimpleXMLElement */
+    public function addAttribute(string $name, string $value = null, string $ns = null) {}
+
+    /** @return string */
+    public function getName() {}
+
+    /** @return string */
+    public function __toString() {}
+
+    /** @return int */
+    public function count() {}
```

Comment:
Here I think we can 
```suggestion
    public function count(): int {}
```

But I'm not sure.